website/docs/reference/warehouse-setups/snowflake-setup.md

```diff
@@ -212,6 +216,7 @@ The "base" configs for Snowflake targets are shown below. Note that you should a
 | retry_on_database_errors | No | A boolean flag indicating whether to retry after encountering errors of type [snowflake.connector.errors.DatabaseError](https://github.com/snowflakedb/snowflake-connector-python/blob/ffdd6b3339aa71885878d047141fe9a77c4a4ae3/src/snowflake/connector/errors.py#L361-L364) |
 | connect_retries | No | The number of times to retry after an unsuccessful connection |
 | connect_timeout | No | The number of seconds to sleep between failed connection retries |
+| reuse_connections | No | A boolean flag indicating whether to close connections after node execution or to reduce total connections opened by reusing idle ones. Default: False |
```

Comment:
suggestion: `A boolean flag indicating whether to reuse idle connections to help reduce total connections opened`